### PR TITLE
ECL-597: Improve test coverage

### PR DESCRIPTION
**Pull request by liandra-giacomine**

### CODE REVIEW

File: app/uk/gov/hmrc/economiccrimelevyaccount/connectors/DesConnector.scala [hunk 1]
```diff
@@ -54,7 +49,8 @@ class DesConnector @Inject() (
     retryFor[ObligationData]("DES - obligation data")(retryCondition) {
       httpClient
         .get(url"${desUrl(eclRegistrationReference.value)}")
-        .transform(_.addHttpHeaders(desHeaders: _*))
+        .setHeader((HeaderNames.authorisation, s"Bearer ${appConfig.desBearerToken}"))
+        .setHeader((CustomHeaderNames.environment, appConfig.desEnvironment))
```
Comment (nemanjatomanovic) [inline app/uk/gov/hmrc/economiccrimelevyaccount/connectors/DesConnector.scala:53]: Why did you change transform with `desHeaders` method into separate calls to set headers ?

Comment (liandra-giacomine) [reply app/uk/gov/hmrc/economiccrimelevyaccount/connectors/DesConnector.scala:53]: So that we could test that they are getting sent out, it wasn't possible with transform

**Review by nemanjatomanovic — APPROVED** [review]
